src/mapml.css

```diff
@@ -63,6 +56,23 @@
      -moz-appearance: none;
           appearance: none;
 }
+
+/* Remove opinionated link colors set by Leaflet. */
+.leaflet-container a {
+  color: revert;
+}
+/* Don't use UA styles for links that are controls
+  (and don't inherit color from map into the reload button for consistency). */
+.mapml-reload-button,
+.leaflet-control-container a,
+.leaflet-container a.leaflet-popup-close-button,
+.leaflet-container a.leaflet-popup-close-button:hover {
+  color: #000;
+}
+/* Revert above `color: #000` for legendlinks. */
+.mapml-layer-item-name a {
+  color: revert;
+}
```

Comment:
FYI: We wouldn't need most of these (and in other places where we "fix" Leaflet styles) if we could just edit the source leaflet.css file - in this case to simply remove the opinionated color they apply to links - but that's got its own set of problems I suppose...

Reply:
Managing styles would also be simpler if Leaflet just used buttons for buttons, and not links for buttons.